agent/connect/ca/testing.go

```diff
@@ -263,3 +263,11 @@ func ApplyCARequestToStore(store *state.Store, req *structs.CARequest) (interfac
 		return nil, fmt.Errorf("Invalid CA operation '%s'", req.Op)
 	}
 }
+func requireTrailingNewline(t testing.T, leafPEM string) {
```

Comment:
very minor: I missed this before

```suggestion
func requireTrailingNewline(t testing.T, leafPEM string) {
t.Helper()
```